nilearn/region.py

```diff
@@ -9,6 +9,7 @@
 
 import numpy as np
 from scipy import linalg, ndimage
+from six.moves import xrange
```

Comment:
No problem, it's a huge diff that you've gone over a few times... really thanks!